src/client/scenes/Admin/Admin.js

```diff
@@ -25,18 +32,26 @@ class Admin extends React.Component {
   };
 
   updateTab = name => {
-    hashHistory.push(`admin/${name}`);
     this.setState({
       tab: name
     });
+    if (name != this.getCurrentTab()) {
+      this.props.history.push(`/admin/${name}`);
+    }
   };
 
   render() {
     return (
       (this.state.loading && <Loader />) ||
       <div>
         <AdminNavBar handleSelect={this.updateTab} activeTab={this.state.tab} />
-        {this.props.children}
+        {/*{this.props.children}*/}
+        { (this.props.location.pathname === '/admin') ? <Redirect to={`${this.props.match.url}/users`} /> : ""}
```

Comment:
You could save yourself SO many keystrokes by picking all this from `this`, before hand:

```
render() {
    const {location, match} = this.props;
    // ....
    <Route path={`${match.url}/topics`} component={ManageTopics} />
   // ....
}
```